_build.ts

```diff
@@ -11,17 +8,20 @@ packageJson.version = Deno.args[0];
 await build({
   entryPoints: ["./index.ts"],
   outDir: "./npm",
-  typeCheck: false,
   test: false,
   scriptModule: false,
   compilerOptions: {
     target: "ES2021",
     sourceMap: true,
     inlineSources: true,
   },
-  shims: {},
+  shims: {
+    deno: true,
```

Comment:
Why do we need to shim Deno?

Reply:
Oops. This was a remnant from setting `test: true`, in which case it's required to get the tests to run. I'll take it out.